iotdb-core/datanode/src/main/java/org/apache/iotdb/db/service/metrics/FileMetrics.java

```diff
@@ -114,49 +135,20 @@ public void bindTo(AbstractMetricService metricService) {
   }
 
   private void bindTsFileMetrics(AbstractMetricService metricService) {
-    metricService.createAutoGauge(
-        Metric.FILE_SIZE.toString(),
-        MetricLevel.CORE,
-        this,
-        o -> o.getFileSize(true),
-        Tag.NAME.toString(),
-        "seq");
-    metricService.createAutoGauge(
-        Metric.FILE_SIZE.toString(),
-        MetricLevel.CORE,
-        this,
-        o -> o.getFileSize(false),
-        Tag.NAME.toString(),
-        "unseq");
     metricService.createAutoGauge(
```

Comment:
We can add that mods files tend to be small, so the benefit of expanding to region metric is too small, so we'll leave it at that